test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -140,9 +140,9 @@ def func(x, b=torch.tensor(1.0)):
             func, (tensor_x,), {"b": torch.tensor(5.0)}
         )
 
-    @pytorch_test_common.xfail(
+    @pytorch_test_common.skip_dynamic_fx_test(
```

Comment:
I thought static also fail for this, when tested with additional test inputs?

Reply:
Additional test inputs are only used when the model is dynamic. For this case, when the model itself is static (the model actually can only be static.), the constant input of nodes should not change. If we expect the constant input can be changed, then the test should be dynamic test. The whole case is a weird case though.

Reply:
I think the dynamic-static concept only matters in input shapes?

Reply:
I see. My thinking was like "int is static, and symint is dynamic." But back to this case, when dynamic_shape=False, this would work, but of course it wouldn't work if that non-tensor input was changed, which doesn't exist, as additional_test_input only serves dynamic_tests.

I can free that condition, and make this back to xfail.

Reply:
@BowenBao, I have a second thought on this. Do we want `additional_test_input` used on static? If that's the case, I will need to move `if self.dynamic_shapes` to test cases to prevent static shape cases from breaking by this change, because rn we are using it to validate dynamic shapes.

Reply:
I modified the goal of this test. LMK if it's satisfactory!